content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -239,19 +242,26 @@ status:
 ...
 ```
 
-The new Pod conditions must comply with Kubernetes [label key format](/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set).
-Since the `kubectl patch` command still doesn't support patching object status,
-the new Pod conditions have to be injected through the `PATCH` action using
-one of the [KubeClient libraries](/docs/reference/using-api/client-libraries/).
+The Pod conditions you add must have names that meet the Kubernetes [label key format](/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set).
+
+
+### Status for Pod readiness {#pod-readiness-status}
 
-With the introduction of new Pod conditions, a Pod is evaluated to be ready **only**
-when both the following statements are true:
+The `kubectl patch` command does not support patching object status.
+To set these `status.conditions` for the pod, applications and
+{{< glossary_tooltip term_id="operator-pattern" text="operators">}} should use
+the `PATCH` action.
+You can use a [Kubernetes client library](/docs/reference/using-api/client-libraries/) to
+write code that sets custom Pod conditions for Pod readiness.
+
+For a Pod that uses custom conditions, that Pod is evaluated to be ready **only**
+when both the following statements apply:
```

Comment:
nit: Check wording:
when both of the following statements apply:
OR
only when the following statements apply: